pkg/controller/nodeconfig/sync.go

```diff
@@ -254,26 +283,55 @@ func (ncc *Controller) sync(ctx context.Context, key string) error {
 
 	progressingConditions := controllerhelpers.FindStatusConditionsWithSuffix(statusConditions, scyllav1alpha1.ProgressingCondition)
 
-	// Use a default progressing node condition for nodes which haven't propagated their conditions to status yet.
-	nodeProgressingConditionTypeFunc := func(node *corev1.Node) string {
-		return fmt.Sprintf(internalapi.NodeProgressingConditionFormat, node.Name)
+	// Use default controller node progressing conditions for nodes which haven't propagated their conditions to status yet.
```

Comment:
The "default" condition is called `Unknown` and the nodeconfig controller should fill in such condition for every node that haven't reported it yet (not just the aggregate). `nodeconfig.calculateStatus` seems like a good place to do it. With the existing condition present in the status, the aggregation may get easier as well.

Reply:
Are you saying the NodeConfig controller should actually set the missing conditions in status? I wanted to avoid having any intersections between the conditions being set by NodeConfig and NodeTune controllers, the filled in defaults I'm only using for aggregation

Reply:
I don't see the conflict. It's the same resource (RV), so no extra conflicts. Each condition has it's own observedGeneration.

From the user's point of view, there should be info about the condition even if in Unknown state. Say the pod can fail to start, or get blocked on resources and the field would be missing for a long time.

Reply:
done